uwblueprint/headsupguys
# feat: Certificate completion page

## Notion ticket link

[Certificates: End of Module Page](https://www.notion.so/uwblueprintexecs/Certificates-End-of-Module-Page-f378628b944e45eaaeee37937d223a3e)



## Implementation description
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/41309709/165371323-a46f311f-c039-4915-8179-b899ed2aec78.png">

* Implements the `certificate/[toolId]` page, uses dynamic data
* Hooks up the following buttons/links
  * Toolkit > Tool breadcrumb
  * Exit and Save Progress
  * Download
  * Back to Toolkit
  * Remaining button: Share
* Adds top bar (with HUG logo linked to home) to Module page and this certificate page
* Final slide ('next' button forced to 'finish') links to Certificate page

https://www.loom.com/share/e26f328b6b374d228b14f5c44ee2c705

## Steps to test
2. Visit http://localhost:3000/
3. Ensure you are logged in, if not log in at http://localhost:3000/login
4. Ensure a tool with a module exists, if not create one at http://localhost:3000/admin/dashboard/modules
5. Click into a module and go to final slide
6. Click Finish, check if the certificate page pops up with correct info
7. Test the buttons listed above, see if they work as intended



## What should reviewers focus on?
* Make sure the page is styled correctly and functional in terms of its buttons


## Checklist
